src/color/color_utils.js

```diff
@@ -79,14 +81,15 @@ define(function(require) {
   /**
    * For a color expressed as an RGBA array, return the corresponding HSBA value
    * 
-   * @param {Array} rgba An 'array' object that represents a list of
-   *                          RGB colors on a scale of 0-255
-   * @return {Array} an array of HSB values, on a scale of 0-255
+   * @param {Array} rgba An 'array' object that represents a list of RGB colors
+   * @param {Array} maxes An 'array' object that represents the RGB range maxes
+   * @return {Array} an array of HSB values, scaled by the HSB-space maxes
    */
-  p5.ColorUtils.rgbaToHSBA = function(rgba) {
-    var var_R = rgba[0]/255;
-    var var_G = rgba[1]/255;
-    var var_B = rgba[2]/255;
+  p5.ColorUtils.rgbaToHSBA = function(rgba, maxes) {
+    var var_R = rgba[0]/maxes[0];
+    var var_G = rgba[1]/maxes[1];
+    var var_B = rgba[2]/maxes[2];
+    var var_A = rgba[3]/maxes[3];
```

Comment:
thinking about this a little more though, I am realizing that perhaps without any clear guide of how code/style reviews are supposed to happen (by who, where, at what point, to what extent), they could potentially feel discouraging or unfriendly even when done with the best of intentions.

paging @workergnome who is in the process of writing up the workflow for contributions. I think we could potentially ease things by setting clearer expectations around code reviews and merging. I'd love to help think through this, once we have a draft of the workflow we could add a section about this, and maybe @sarahgp and @kadamwhite you'd be willing to help?